source/Engine/Application.cpp

```diff
@@ -1265,16 +1287,18 @@ void Application::PollEvents() {
 				break;
 			}
 
-			if (DevMenu) {
-				// Quit game (dev)
+			if (DevMode) {
+				// Open dev menu (dev)
 				if (key == KeyBindsSDL[(int)KeyBind::DevQuit]) {
```

Comment:
I strongly suggest you make a separate key bind for the dev menu (otherwise "quit" won't actually quit when the dev menu is enabled, which is misleading.)